### DISCUSSION TITLE
Lettuce async

### DISCUSSION THREAD

**Discussion by gusah009** [BDD-CLUB/challenge-every-week-posting#385] [1. 개발 관련 글 모음]
java redis client 중 하나인 lettuce의 async 버전을 사용하고 공부해봤습니다.

https://gusah009.github.io/etc/2024-03-20-Lettuce/

**Comment by hyun98** [comment]
사연이 많아보이는 글이네요

**Reply by gusah009** [reply]
눈물이 앞을 가리네요

**Comment by jojaeng2** [comment]
redis 라이센스가 바뀐다던데요

**Reply by gusah009** [reply]
다른 인메모리 캐시가 유행하면 또 공부해야겠네요
